.gitignore

```diff
@@ -8,3 +8,4 @@ Gemfile.lock
 pkg/*
 coverage/*
 TODO
+.DS_store
```

Comment:
That's unrelated to this PR (should be in `~/.gitignore` anyway)

Reply:
Sorry!